backend/src/main/java/com/cs471/studentLoanSystem/login/LoginSystem.java

```diff
@@ -0,0 +1,43 @@
+package com.cs471.studentLoanSystem.login;
+
+import java.net.URI;
+
+import com.cs471.studentLoanSystem.roles.StudentInfo;
+import org.springframework.http.ResponseEntity;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+@RestController
+public class LoginSystem {
+
+    @GetMapping("/login")
+    public String loginForm(Model model) {
+        model.addAttribute("LoginInfo", new LoginInformation());
+        return "LoginInfo";
+    }
+
+    @PostMapping("/login")
+    public ResponseEntity<LoginResponse> loginForm(
```

Comment:
I haven't quite seen this done this way before. Should `Model model` be removed since this shouldn't be generating an HTML page? Also, I believe in order to map in a JSON request, you will need `@ModelAttribute` to be replaced with `@ResponseBody`

Reply:
Ignore this. Tested on your PC with SoapUI. It returns JSON. Happy Andy is happy.